source/manual/view-extract-feedback-data-bigquery.html.md.erb

```diff
@@ -0,0 +1,150 @@
+---
+owner_slack: "#govuk-data-labs"
+title: View or extract feedback data in BigQuery
+section: Data Community Documentation
+layout: manual_layout
+parent: "/manual.html"
+---
+
+You can do the following with the feedback data in BigQuery:
+
+- use Google Data Studio to present the data
+- extract the data from BigQuery into another file format, for example a CSV or a Google Sheets file
+
+You do not need a specific permission to view or extract feedback data in BigQuery once you have access to the BigQuery data sets.
```

Comment:
how to get access to BigQuery data sets? ask data labs

Reply:
will update